abstract/group.go

```diff
@@ -51,6 +51,9 @@ type Secret interface {
 
 	// Set to a fresh random or pseudo-random secret
 	Pick(rand cipher.Stream) Secret
+
+	// SetBytes will take bytes and create a secret out of it
+	SetBytes([]byte) Secret
```

Comment:
No, I'm pretty sure big.Int.Bytes() just returns a *new* slice into which it copies a representation of the integer - and that's what nist.Int.Bytes() should do as well.  Note that big.Int has a Words() function that actually returns the internal-representation slice, but that's a slice of (32-bit?) words instead of bytes for efficiency, and it's only intended for use in performance-critical extensions to big.Int that really need to manipulate the internal state efficiently.  That's definitely not what Bytes() is for.  Just create a new slice, encode into it, and return it.